Added section "The Booster Way" to the README.md file (#488)
boostercloud/booster: README.md

* *True Serverless*: Serverless is about stop caring about your servers, but many implementations still require long YAML files to describe your infrastructure, and you really need to know what you're doing. True Serverless means that you don't even care about cloud configuration, Booster will figure it out for you based on the code you write.
* *Convention over Configuration:* We prefer to provide  standardized highly-opinionated modules than highly-configurable ones. This helps us to keep your code small and follow the best security and structure practices when deploying your applications to the cloud. High consistency in your project and code structure also helps to abstract out most of the boilerplate.
* *Don't Repeat Yourself (Extreme edition):* /The only code that matters is the one that makes your application different/. We push TypeScript structure and type system to the limit to avoid the need of writing repetitive code, like object-to-JSON serializations, API or database schemas or redundant architecture layers. 

> * *Self-documenting APIs* We adopted GraphQL because it's a self-documenting standard. You can grab ApolloClient and start using a Booster backend in a minute. We try to be as close as that with any other integration.

Reviewer: I would omit the specific GraphQL client and I would just sugfest to grab any GraphQL client

Reviewer 2: I partially agree, as the websockets protocol we've implemented is the one used by Apollo, but at the same time, here in the docs it doesn't matter...

Author: ```suggestion
* *Self-documenting APIs* We adopted GraphQL because it's a self-documenting standard. You can grab a standard GraphQL client like [ApolloClient](https://github.com/apollographql/apollo-client) and start using a Booster backend in a minute.
```

> * *Developer's productivity:* Software development is fun, and a modern tool should make it even funnier, reducing the effort needed for mundane tasks. Booster provides code generators to help you quick start new projects and objects, and the framework types and APIs are hand-crafted to help your IDE to help you.

Reviewer 2: Funnier is something that you can laugh at, imo

```suggestion
* *Developer's productivity:* Software development is fun, and a modern tool should make it even more, reducing the effort needed for mundane tasks. Booster provides code generators to help you quick start new projects and objects, and the framework types and APIs are hand-crafted to help your IDE to help you.
```

Author: On the Rails homepage, they say "more fun" https://rubyonrails.org

Reviewer 2: Then let's say more fun, but not funnier :)

Author: ```suggestion
* *Developer's productivity:* Software development is fun, and a modern tool should make it even more fun, reducing the effort needed for mundane tasks. Booster provides code generators to help you quickstart new projects and objects, and the framework types and APIs are hand-crafted to help your IDE help you.
```
